spec/features/new_project_spec.rb

```diff
@@ -111,12 +105,6 @@
     expect(locales_en_file).to match(/application: #{app_name.humanize}/)
   end
 
-  scenario "config simple_form" do
-    run_suspenders
-
-    expect(File).to exist("#{project_path}/config/initializers/simple_form.rb")
-  end
-
```

Comment:
Instead of removing these tests, let's enable the required gem inside the test and make sure it still works.